File: FirstApp/FormView/QrCodeView.vb
Imports QRCoder

Public Class QrCodeView
    Private Sub QrCodeView_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        If Tag IsNot Nothing Then
            QrNew(Tag)
        End If
    End Sub

    Public Sub QrNew(qr As String)
        QrBox.Image = Nothing
        Dim gen As New QRCodeGenerator
        Dim data = gen.CreateQrCode(qr, QRCodeGenerator.ECCLevel.Q)
        Dim code As New QRCode(data)
        QrBox.Image = code.GetGraphic(10)
    End Sub

End Class

File: OrnaTrack/FormView/QrCodeView.cs
using System;
using Microsoft.VisualBasic.CompilerServices;
using QRCoder;

namespace FirstApp
{

    public partial class QrCodeView
    {
        public QrCodeView()
        {
            InitializeComponent();
        }
        private void QrCodeView_Load(object sender, EventArgs e)
        {
            if (Tag is not null)
            {
                QrNew(Conversions.ToString(Tag));
            }
        }

        public void QrNew(string qr)
        {
            QrBox.Image = null;
            var gen = new QRCodeGenerator();
            var data = gen.CreateQrCode(qr, QRCodeGenerator.ECCLevel.Q);
            var code = new QRCode(data);
            QrBox.Image = code.GetGraphic(10);
        }

    }
}